src/main/kotlin/com/matthewrussell/trwav/CuePoint.kt

```diff
@@ -1,6 +1,44 @@
 package com.matthewrussell.trwav
 
+import java.util.regex.Pattern
+
 data class CuePoint(
     var location: Int = 0,
     var label: String = ""
-)
\ No newline at end of file
+): Comparable<CuePoint> {
+    override fun compareTo(other: CuePoint): Int {
+        // Sort Cues by location first then by Label
```

Comment:
any particular reason why you would care about the label? It would be probably better to just ignore it since there's a chance it doesn't have integer data in it.

The only time you would fall back to sorting by the label here is if multiple verse markers have the exact same location. I would expect this to be a rare event, but also likely a problem if there are two with the same location to begin with.

Reply:
@jsarabia That's right in tr-wav sorting algorithm is redundant. I removed it.